lib/ui/platform_dispatcher.dart

```diff
@@ -73,6 +73,23 @@ const String _kFlutterKeyDataChannel = 'flutter/keydata';
 ByteData? _wrapUnmodifiableByteData(ByteData? byteData) =>
     byteData == null ? null : UnmodifiableByteDataView(byteData);
 
+/// A token that represents a root isolate.
+class RootIsolateToken {
+  RootIsolateToken._(this._rootIsolateId);
+
+  final int _rootIsolateId;
```

Comment:
Here and elsewhere: please avoid calling this the "isolate id". The VM service already has a concept of isolate IDs that will certainly be different than this number. Someone a couple years from now will waste a lot of time trying to figure out why these are different.

Maybe just call it `_token` or something. I'd also suggest documenting here that this is a raw pointer to an engine-owned C++ object, and that the Dart VM itself does not use or know about this number.

Reply:
Done.

Reply:
I don't want to comment about it being a raw pointer.  It's just an implementation detail and I don't want to guarantee it will always be that.  We could implement it differently, just a static int64_t that increments for example.

Reply:
Fair, maybe just a comment about it not being something based on Dart or the VM service?

Reply:
I added a comment to the field.  "Root isolate" is a Flutter specific term, so that should denote it's a Flutter concept, right?